go.mod

```
@@ -1,6 +1,6 @@
 module github.com/mongodb/terraform-provider-mongodbatlas
 
-go 1.20
+go 1.21.3
```

```suggestion
go 1.21
```
if you want to specify a patch use [toolchain](https://go.dev/doc/toolchain) (this is a new feature for 1.21)

anecdotally I just fixed this issue in the Atlas operator https://github.com/mongodb/mongodb-atlas-kubernetes/pull/1188

---

uhm, i remove the line and "go mod tidy" did this for me

---

changed